schema/core/reusable/file_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "schemaId": "reusable-file-metadata",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "file_metadata",
+    "properties": {
+        "pathname": {
+            "description": "Path to the directory that contains the file",
+            "type": "string"
+        },
+        "basename": {
+            "description": "Basename of the file",
+            "type": "string"
+        },
+        "filetype": {
```

Comment:
We should limit the filetype values in the `file_to_display` (to be named `file_content` property)

Reply:
Filetype values are now restricted (in `file_content`) to be either `image` or `text`